index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Porfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav>
+      <p>Toyin</p>
+      <ul>
+        <li><a href="">About</a></li>
+        <li><a href="">Porfolio</a></li>
+        <li><a href="">Contact</a></li>
+      </ul>
+    </nav>
+    <header>
+      <div class="header-background"></div>
+      <div class="header-text">
+        <p>
+          Hi! My name is Toyin Ayorinde, I'm a front end developer from Lagos
+          Nigeria
+        </p>
+      </div>
+    </header>
+    <main>
+      <div class="about">
+        <div class="about-left">
+          <img src="./images/profile.JPG" alt="" />
+        </div>
+        <div class="about-right">
+          <hr />
+          <h1>About me</h1>
+          <p>
+            I'm a sotfware developer with experience in creating applications
+            using HTML, CSS and JavaScript. I like learning about other
+            platforms such as the backend because it gives a wider view on how
+            to approach problems. I have a good understanding of version control
+            and team collaboration tools such as Pivotal Tracker. I spend my
+            spare time reading, playing basketball, swimming or generally just
+            having fun. I have a keen interest in languages, currently I speak
+            English, French, Hausa and Yoruba. In the coming years I plan to
+            increase the list with probably Spanish. If you want to work with
+            me, feel free to <a href="contact.html">get in touch</a>
+          </p>
+          <a href="portfolio.html"><button>Go to portfolio</button></a>
```

Comment:
Always use <a> for links and <button> for JavaScript events.

Reply:
Done